addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -435,7 +435,7 @@
             </t>
         </template>
         <template id="price_dynamic_filter_template_product_product" name="Dynamic Product Filter Price">
-            <t t-if="data['prevent_zero_price_sale']">
+            <t t-if="data.get('prevent_zero_price_sale', True)">
```

Comment:
I'd put False as fallback personally :thinking:

Reply:
There is no price in the sample data?
Because I'd find the sample more beautiful if it was displaying a fake price and button instead of the 'zero price sale' message, regardless of the website configuration